### PR TITLE
code reformatting and update unit tests to use more specific asserts

### PR DESCRIPTION
**Pull request by hugovk**
* Upgraded Python syntax with [`pyupgrade --py37-plus`](https://github.com/asottile/pyupgrade)

* Upgraded unit tests to use more useful asserts

* `sudo` is no longer needed in `.travis.yml`
   * https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration

### CODE REVIEW

**Comment by tzarebczan** [comment]
Thanks for the PR! We'll get you a review soon. Can we show you some [appreciation](https://lbry.com/faq/appreciation) for the contribution?

Make sure to check out https://lbry.tech/contribute and sign up for the dev mailing list at the bottom!

File: torba/torba/workbench/_blockchain_dock.py [hunk 1]
```diff
@@ -1,5 +1,3 @@
-# -*- coding: utf-8 -*-
-
 # Form implementation generated from reading ui file 'blockchain_dock.ui',
 # licensing of 'blockchain_dock.ui' applies.
```
Comment (eukreign) [inline torba/torba/workbench/_blockchain_dock.py]: This file is generated and should not be edited.

File: torba/torba/workbench/_output_dock.py [hunk 2]
```diff
@@ -1,5 +1,3 @@
-# -*- coding: utf-8 -*-
-
 # Form implementation generated from reading ui file 'output_dock.ui',
 # licensing of 'output_dock.ui' applies.
```
Comment (eukreign) [inline torba/torba/workbench/_output_dock.py]: This file is generated and should not be edited.

File: torba/torba/server/util.py [hunk 3]
```diff
@@ -182,7 +182,7 @@ class LogicalFile:
     """A logical binary file split across several separate files on disk."""
 
     def __init__(self, prefix, digits, file_size):
-        digit_fmt = '{' + ':0{:d}d'.format(digits) + '}'
+        digit_fmt = '{' + f':0{digits:d}d' + '}'
```
Comment (eukreign) [inline torba/torba/server/util.py]: Can the `'{'` and `'}'` go inside of the f-string?

File: torba/torba/server/script.py [hunk 4]
```diff
@@ -51,13 +51,13 @@ def __init__(self, name, enumList):
...
+                raise EnumError(f"enum name {x} not a string")
             if not isinstance(i, int):
-                raise EnumError("enum value {} not an integer".format(i))
+                raise EnumError(f"enum value {i} not an integer")
             if x in uniqueNames:
-                raise EnumError("enum name {} not unique".format(x))
+                raise EnumError(f"enum name {x} not unique")
             if i in uniqueValues:
-                raise EnumError("enum value {} not unique".format(x))
+                raise EnumError(f"enum value {x} not unique")
```
Comment (eukreign) [inline torba/torba/server/script.py]: Should that be `{i}` instead of `{x}`?

Comment (hugovk) [reply torba/torba/server/script.py]: I think so, looks like a bug in the original too. Shall I change it here?

File: torba/torba/workbench/_blockchain_dock.py (hunk 1 above)
Comment (hugovk) [reply torba/torba/workbench/_blockchain_dock.py]: Reverted!

File: torba/torba/workbench/_output_dock.py (hunk 2 above)
Comment (hugovk) [reply torba/torba/workbench/_output_dock.py]: Reverted!

File: torba/torba/server/util.py (hunk 3 above)
Comment (hugovk) [reply torba/torba/server/util.py]: Yes, updated. In f-strings, curly braces are escaped by doubling them: https://stackoverflow.com/a/5466478/724176.

File: torba/torba/server/script.py (hunk 4 above)
Comment (eukreign) [reply torba/torba/server/script.py]: yes

**Comment by hugovk** [comment]
@tzarebczan Hi! What do I need to apply for the Hacktoberfest tshirt? My other PR is https://github.com/lbryio/lbry-sdk/pull/2499. Thank you!

**Comment by tzarebczan** [comment]
The promotion is now over, but you can reach out per the original instructions and we'll get some LBRY credits over to you for the work.

Link: https://lbry.com/faq/appreciation

**Comment by hugovk** [comment]
Oh, that's a bit disappointing because your https://lbry.com/news/hacktoberfest-2019 didn't say how to apply, and doesn't give a deadline for when to apply. I was being patient waiting to hear from your side. Never mind then!

**Comment by tzarebczan** [comment]
Sorry for the trouble. We'll see if there are any extras shirts or stickers.

The call to action was here: https://github.com/lbryio/lbry-sdk/pull/2498#issuecomment-537764646

**Comment by hugovk** [comment]
Thank you! I did check that link at the time, but it only mentions LBC and not Hacktoberfest.

**Pull request merged**